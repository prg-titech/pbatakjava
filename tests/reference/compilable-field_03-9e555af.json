{
    "basename": "compilable-field_03-9e555af",
    "cmd": "{infile}",
    "infile": "tests/field_03/build.sh",
    "infile_hash": "7e2ac6a407268835af55c8490e5fbd0e9e5d6c92a2a381cd98bf27e7",
    "stdout": "compilable-field_03-9e555af.stdout",
    "stdout_hash": "1733c0bacec01fd8daf69013a519c57fe78712b91dd64ea1124f45ca",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}