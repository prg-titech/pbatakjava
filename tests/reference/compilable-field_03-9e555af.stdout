
public interface C {
  public int get__x();

  public int set__x(int toAssign);
}

public interface C_Factory {
}

public class C_v1 implements C {
  public int x;

  public int get__x() {
    return this.x;
  }

  public int set__x(int toAssign) {
    return this.x = toAssign;
  }
}

public class C_v1_Factory implements C_Factory {
}

public interface D {
  public float get__x__float();

  public float set__x(float toAssign);

  public C m();
}

public interface D_Factory {
}

public class D_v1 extends C_v1 implements D {
  public float x;

  public C m() {
    return (C_v1) this;
  }

  public float get__x__float() {
    return this.x;
  }

  public float set__x(float toAssign) {
    return this.x = toAssign;
  }
}

public class D_v1_Factory implements D_Factory {
}
